lib/provider/aws/lambda-edge-origin-request/sanitize-headers.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const setCookieVariations = require('./set-cookie.json').variations;
+const readOnlyHeaders = [
+  "memoept-encoding",
```

Comment:
Typo: fixed in the code block [I've suggested here](https://github.com/dougmoscrop/serverless-http/pull/192/files/5b6429d0cd893adad9b131289c49cc4f7c851d2b#r553700434)